packages/template-retail-react-app/app/ssr.js

```diff
@@ -29,7 +29,12 @@ const options = {
     // The protocol on which the development Express app listens.
     // Note that http://localhost is treated as a secure context for development,
     // except by Safari.
-    protocol: 'http'
+    protocol: 'http',
+
+    // Option for whether to set up a special endpoint for handling
+    // private SLAS clients
+    // Set this to false if using a SLAS public client
+    useSlasPrivateClient: false
```

Comment:
Its worth thinking hard about a) the name we use here ... this seems pretty good, only nit would be `useSLASPrivateClient` b/c SLAS is already an acronym and b) the code comment we use to describe this.

If setting this to true _requires_ an environment variable to exist, it would be useful to mention it.